cilium/CFP-39876-clustermesh-filtered-export.md

```diff
@@ -0,0 +1,156 @@
+
+## Goals
+
+- Enhance Cilium Clustermesh scalability through selective endpoint and identity distribution
+- Network policies for resources under global namespaces
+- Endpoint to Endpoint connectivity for resources under global namespaces
+- Global service functionality for resources under global namespaces
+- Support scoped export for CRD mode identities
+
+## Non-Goals
+
+- Network policies for non global(local) namespaces
+- Endpoint to Endpoint connectivity for non global(local) namespaces
+- Global service functionality for non global(local) namespaces
+- Scoped export for non CRD modes of identity allocation 
+ 
+## Network Policy Support 
+# Cross-Cluster Communication and Network Policy Behavior
+
+Full support for **cross-cluster communication** and **network policy enforcement** is available **only when both the source (client) and destination (server) pods**—which may optionally be accessed via a service—**are part of a _global namespace_**.
+
+### Behavior When Pods Are Not in Global Namespaces
+
+If either the source or the destination pod does **not** belong to a global namespace, cross-cluster communication **may still succeed**, but under certain conditions although we do not support as part of the new mode. The evaluation of network policy for different cases is summarize below:
+
+
+### Summary of Enforcement Logic
+
+| Source Pod in Global NS | Destination Pod in Global NS | Network Policy Applies | Traffic Allowed? |
+|-------------------------|-------------------------------|-------------------------|------------------|
+| Yes                     | Yes                           | Yes                     | ✅ Enforced normally |
+| No                      | Yes                           | Ingress Policy          | ❌ Works for tunnel mode. Does not work for native routing mode  |
+| Yes                     | No                            | Egress Policy           | ❌ Works for tunnel mode. Does not work for native routing mode |
+| No                      | No                            | Any Policy              | ❌ Works for tunnel mode. Does not work for native routing mode  |
+| Any                     | Any                           | No Policies             | ✅ Allowed (no restrictions) |
+
+Even though the policies work for tunnel routing mode, for consistency in customer experience, we do not provide support for network policies for workloads outside global namespaces. 
+
+## Global Service Support
+In the scoped-export mode, all services within global namespaces would automatically be marked as "global". Customers are not required to explicitly annotate individual services. Also, a service which is annotated as global but does not reside in a global namespaces would no longer be global.  
```

Comment:
I am not sure did you removed details about scoped-export in the ConfirMap but this option is still supposed to exist? If so could you still detail that you are going to add it in the ConfigMap?